src/sentry/lang/javascript/processor.py

```diff
@@ -381,6 +391,21 @@ def fetch_file(url, project=None, release=None, dist=None, allow_scraping=True):
     return result
 
 
+def get_max_age(headers):
+    cache_control = 'cache-control'
+    max_age = CACHE_CONTROL_MIN
+
+    if has_header(headers, cache_control):
+        match = re.search(CACHE_CONTROL_RE, headers[cache_control])
+        if match:
+            max_age = max(CACHE_CONTROL_MIN, int(match.group(1)))
+    return min(max_age, CACHE_CONTROL_MAX)
+
+
+def has_header(headers, header_name):
+    return header_name.lower() in {header.lower() for header in headers}
```

Comment:
Aren't the headers here case insensitive? iirc this is some case insensitive dict backing the values, so we shouldn't have to do this work.

Reply:
Correction for future reference: They are set to all lower case as seen here https://github.com/getsentry/sentry/blob/master/src/sentry/http.py#L356